cmd/keytransparency-signer/main.go

```diff
@@ -86,17 +79,12 @@ func main() {
 	tmap := trillian.NewTrillianMapClient(mconn)
 
 	// Connection to append only log
-	tlog, err := config.LogClient(*logID, *logURL, *logPubKey)
+	lconn, err := grpc.Dial(*logURL, grpc.WithInsecure())
```

Comment:
The trillian log and map are not always on the same machine.

Using `grpc.WithInsecure()` is not a change.  It was previously hidden behind `config`.  This makes it a bit more obvious and consistent.